src/createServer.js

```diff
@@ -1,3 +1,29 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { getRequestParams } = require('./getRequestParams');
+const { validateRequestParams } = require('./validateRequestParams');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const params = getRequestParams(request);
+    const errors = validateRequestParams(params);
+
+    if (errors.errors.length) {
+      response.statusCode = 400;
+      response.end(JSON.stringify(errors), 'utf-8');
```

Comment:
<img width="654" alt="image" src="https://github.com/mate-academy/node_case-transformer-server/assets/86789161/155911f5-e321-4af8-b15c-0b9b0174d13b">

statusText also should be provided

Reply:
![image](https://github.com/mate-academy/node_case-transformer-server/assets/26518146/5075b14f-aee9-4d1c-917d-72c5f1c8cf5c)
`Bad request` is the standard message for the status code 400.
![image](https://github.com/mate-academy/node_case-transformer-server/assets/26518146/829c0dcd-a6f9-4856-ad6d-3a1dd29534ba)